### ISSUE TITLE
Do you mind if I hook thi repo up for an open-sourced project?

### ISSUE THREAD

**Issue by erogol** [fatchord/WaveRNN#18]
I plan to merge WaveRNN with TTS - https://github.com/mozilla/TTS, if you don't mind.

**Comment by fatchord** [comment]
@erogol Absolutely. I'd be delighted to contribute in some way to a Mozilla project (love the company's products and philosophy!).  All I'd ask is if you could credit this repo.

**Comment by erogol** [comment]
Thanks happy to hear that. Reference sure thing. I've already forked your work and start training with some changes. I appreciate your work and simplicity of the code.

Then could you please consider to license your repo with one of the open source license, for example MIT?

**Comment by fatchord** [comment]
@erogol I'm not 100% sure on the licensing yet as I might use this algorithm in a commercial product in the future or a derivation thereof. Honestly, I know next to nothing about licensing so I'm not in a position to make an informed decision about such matters right now.

Anyway for the time being, Mozilla has my express permission to use the code in this repo as long as I'm credited.
